src/extension/app/components/spectrum/menu-group/menu-group.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I've created a new folder in `components` called spectrum. The purpose is a place for any spectrum components we are overriding. There are a few existing components we should think about moving here as well.